python/GafferDispatchUI/FrameMaskUI.py

```diff
@@ -64,6 +64,12 @@
 
 		),
 
+		"dispatcher.isolated" : (
+
+			"plugValueWidget:type", "",
+
+		),
```

Comment:
I wonder if we should go further here, and not even add the plug to nodes where it's not relevant? I'm a little scared of a plug getting turned on accidentally by some pipeline script, and it taking a while to debug because it's not even shown in the UI. The other problem is that we're relying on extensions to do the same for their own TaskNodes.

So I wonder if we should make the whole thing opt-in? Perhaps we just add a `dispatcher:allowIsolation` entry to the node metadata registration for the nodes where we want the plug to be added?

Reply:
That does sound good, especially for covering the case of Tasks from extensions. I've added that in https://github.com/GafferHQ/gaffer/pull/6612/commits/9ce45d4b33dbf1f7803feedc2d0b683522540496. I'm doing the registration in `PythonCommand.py` and `SystemCommand.py` for now.

I guess it could go in a `startup` file, but there's only be two entries. Would it be more discoverable though?

> Perhaps we just add a `dispatcher:allowIsolation` entry to the node metadata registration

Do you mean the metadata registration in the `SystemCommandUI.py`? That would cause problems with apps that don't load the UI, right? Or is there another registration that would be better?